source/reference/minio-mc-admin/mc-admin-user-svcacct-remove.rst

```diff
@@ -0,0 +1,84 @@
+.. _minio-mc-admin-svcacct-remove:
+
+================================
+``mc admin user svcacct remove``
+================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin user svcacct remove
+
+
+Syntax
+------
+
+.. start-mc-admin-svcacct-remove-desc
+
+The :mc-cmd:`mc admin user svcacct remove` command removes an access key associated to the specified user.
```

Comment:
You don't actually specify the user in the command, though.
So drop `specified` here and go with
> ... associated to a user on the deployment.